cmd/helm/flags.go

```diff
@@ -24,6 +24,7 @@ import (
 	"sort"
 	"strings"
 
+	"github.com/pkg/errors"
```

Comment:
This package is dead/unmaintained. Please don't add new usages.

Reply:
I will create a pull request to remove it from the codebase

Reply:
Nevermind it's deeply ingrained